src/main.rs

```diff
@@ -282,43 +282,8 @@ struct CheckRelease {
     verbosity: clap_verbosity_flag::Verbosity<clap_verbosity_flag::InfoLevel>,
 }
 
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn verify_cli() {
-        use clap::CommandFactory;
-        crate::Cargo::command().debug_assert()
-    }
-
-    #[test]
-    fn verify_binary_contains_lints() {
-        use assert_cmd::Command;
-
-        let assert_on_crate_pair = |crate_pair: &str| {
-            let mut cmd = Command::cargo_bin("cargo-semver-checks").unwrap();
-            cmd.current_dir(format!("test_crates/{crate_pair}/new"))
-                .args([
-                    "semver-checks",
-                    "check-release",
-                    "--baseline-root=../old/Cargo.toml",
-                ])
-                .assert()
-        };
-
-        // The `template/new` and `template/old` are identical crates, so running cargo-semver-checks on
-        // them shouldn't report any issues.
-        assert_on_crate_pair("template").success();
-
-        // Those test crate pairs should trigger an error (because of a lint with the same name),
-        // so they should return a non-zero exit code.
-        // Only a few (arbitrarily) lints are being checked to speed up the testing process (the full
-        // list of lints is tested in `src/query.rs`).
-        for crate_pair in [
-            "enum_missing",
-            "function_const_removed",
-            "function_unsafe_added",
-        ] {
-            assert_on_crate_pair(crate_pair).failure();
-        }
-    }
+#[test]
+fn verify_cli() {
+    use clap::CommandFactory;
+    Cargo::command().debug_assert()
```

Comment:
Did you mean to change this and pull it out of `mod tests`?

Reply:
I decided to keep `src/main.rs` how it was originally (by simply reverting the `src/main.rs` after my changes to the state before my changes). Should I nevertheless move it to `mod tests`?

Reply:
Sorry, I must have been looking at a stale version of the PR.